src/Core.UnitTests/Binding/NonRoslynBindingConfigFileTests.cs

```diff
@@ -25,34 +25,34 @@
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
-using SonarLint.VisualStudio.Core.CFamily;
+using SonarLint.VisualStudio.Core.Binding;
 
-namespace SonarLint.VisualStudio.Core.UnitTests.CFamily
+namespace SonarLint.VisualStudio.Core.UnitTests.Binding
 {
     [TestClass]
-    public class CFamilyBindingConfigTests
+    public class NonRoslynBindingConfigFileTests
```

Comment:
There was nothing cfamily specific about these classes. They're simply fetching the QP and saving it to disk. The code for js/ts was identical so I renamed them to NonRoslynXXX. 
Unfortunately the Roslyn class is similar but more complicated. I think we can unite parts there as well but I've left it for now.